release-notes.md

```diff
@@ -0,0 +1,45 @@
+# FireCloud Explorer Release Notes
+
+### Alpha version 0.1.0
+
+## Feature summary
+
+This first alpha release of FireCloud Explorer includes the following features:
+
+* Login using FireCloud credentials.
+* Browse and select files associated with FireCloud workspaces.
+* Download selected files to the user’s local file system, optionally preserving the bucket folder structure.
+* Upload selected files from the local file system to a FireCloud workspace that a user can write to.
+* Monitor the status of uploads and download as they proceed.
+
+## Known Issues
+
+The following issues have been identified, but not resolved in this version:
+
+* If there is no internet connection while trying to login the app, the “Choose account” Google popup will appear blank.
+_Temporary solution:​ the user should wait for a few seconds and an alert message will appear notifying the issue, by clicking “Ok”, both Google popup and the
```

Comment:
I believe this will format the output so that the solution is wrapped in the same paragraph as the bullet point on the line before, instead of starting on its own line. Is that OK?

Reply:
I'm looking at the code in an md editor (https://dillinger.io/) and it looks fine

Reply:
Hmm, I'm looking at the file in github, by clicking the View button at the top of this file.

Reply:
:( so what should I add to break the line?

Reply:
well you could use some HTML, like dl/dt:
```
<dl>
  <dt>Definition list</dt>
  <dd>Is something people use sometimes.</dd>
  <dt>Markdown in HTML</dt>
  <dd>Does *not* work **very** well. Use HTML <em>tags</em>.</dd>
</dl>
```